src/components/Board/Board.container.js

```diff
@@ -1185,14 +1185,18 @@ export class BoardContainer extends Component {
     }
   };
 
+  propsHistoryReplace(boardId) {
+    this.props.history.replace(`/board/${boardId}`);
+  }
+
   onRequestPreviousBoard() {
-    if (this.props.navHistory.length >= 2) {
+    this.props.previousBoard();
+    /*if (this.props.navHistory.length >= 2) {
       const prevBoardId = this.props.navHistory[
         this.props.navHistory.length - 2
       ];
-      this.props.history.replace(`/board/${prevBoardId}`);
-      this.scrollToTop();
-    }
+      this.propsHistoryReplace(prevBoardId);*/
+    this.scrollToTop();
```

Comment:
This change look good. Please avoid leaving comments with old lines in the code. Make an atomic commit with a good title to documment what do you made in the change.